lib/cocoapods/sandbox/path_list.rb

```diff
@@ -128,19 +128,21 @@ def relative_glob(patterns, options = {})
         else
           full_list = files
         end
-
-        list = Array(patterns).map do |pattern|
+        patterns_array = Array(patterns)
+        exact_matches = (full_list.to_set & patterns_array.to_set).to_a
```

Comment:
I don't think we need the `to_set` and `to_a` calls

Reply:
Oh yeah, I wouldn't know that array has set intersection operator 👍